src/helpers/validateRoverName.ts

```diff
@@ -0,0 +1,8 @@
+import Rovers from "./roverNames";
+
+export default function validateRoverOrThrow(roverName: string) {
+    const validName = Rovers[roverName.toUpperCase()];
+    if (!validName) {
+        throw new Error("Invalid rover name.");
```

Comment:
It looks like this error will bubble up and return a 500 status code - ideally this would return a 400 (Bad request) status code. No need to fix this now, but worth bearing in mind!